### PR TITLE
[pyqgiscookbook] Add new samples to cheat sheet

### PR DESCRIPTION
**Pull request by All4Gis**
### Description

Add new samples based on my [answer ](https://gis.stackexchange.com/a/327135/49538)from GIS Stack Exchange.

Goal:

Ticket: #

- [ ] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

**Comment by All4Gis** [comment]
I don't understand why on my laptop pass all tests and here not.
Sin título

File: source/docs/pyqgis_developer_cookbook/cheat_sheet.rst [hunk 1]
```diff
@@ -51,6 +51,11 @@ Settings
     for k in sorted(qs.allKeys()):
         print (k)
 
+.. testoutput::
+   :hide:
+
+   qgis/map_update_interval
+
```
Comment (DelazJ) [inline source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: Does this output need to be here? At this point of the doc (tests executed from top to bottom as far as i understand), the `qgis/map_update_interval` setting not yet exists, doesn't it?

Comment (All4Gis) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: Yes I know! But if we don't put it on my laptop, that test fails! it's so strange

Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: Looks weird to me to have it there. Can you remove it and see how it behaves on github?

Comment (All4Gis) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: remove it!

Comment (All4Gis) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: I don't understand it, but in my laptop fails and here pass!

Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: Having the test pass the way it's now makes more sense to me.
> I don't understand it, but in my laptop fails and here pass!

I ru the test locally and it passes... the first time. Then I build the docs with `make html` and now all the calls of `make -f user.mk doctest` fail with recommendation to add the testoutput you had. If that rings a bell for someone...

Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: More: I checkout the master branch (hence removing the setting added in this PR) and building the doctest always fails ( 51 test, 1 failure) with same recommendation. And rebuilding docs changes nothing.

Comment (All4Gis) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: Interesting! Do you think I should add that testoutput again? Or is the problem with travis test?

Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: I'd say "No". I think the issue might reside in our local build though I still don't understand where and I wonder if it'll happen on GitHub later. Anyway the current code is logical and I'd better keep it than doing adjustment that doesn't make sense.
Can you isolate this/these snippet(s) with `.. testcode:: group` and see how the local build behaves?

Comment (All4Gis) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: Using `.. testcode:: group` returns the same error

Comment (All4Gis) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: To pass the tests in local I added `.. testcleanup:: QSettings().clear()` in the **Get QSettings list** sample. I think it's a good option

Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: Then let's try it upstream :D

Comment (All4Gis) [reply source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]: added!

File: source/docs/pyqgis_developer_cookbook/cheat_sheet.rst [hunk 2]
```diff
@@ -190,6 +202,18 @@ Otherwise
...
+**Refresh layer at interval**
+
+.. testcode::
+
+    from qgis.core import QgsProject
+
+    layer = QgsProject.instance().mapLayersByName("layer name you like")[0]
+    # Set seconds (5 seconds)
+    layer.setAutoRefreshInterval(5000)
+    # Enabled auto refresh
```
Comment (DelazJ) [inline source/docs/pyqgis_developer_cookbook/cheat_sheet.rst]:
```suggestion
    # Enable auto refresh
```
?

**Pull request merged**